<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>COwnModelSetForm</class>
 <widget class="QFrame" name="COwnModelSetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="vl_OwnModelSetForm">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_Selection">
     <property name="title">
      <string>Selection</string>
     </property>
     <layout class="QGridLayout" name="gl_Selection">
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_Mode">
        <property name="text">
         <string>Mode:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="rb_Full">
        <property name="toolTip">
         <string>new set, existing data will be deleted</string>
        </property>
        <property name="text">
         <string>full</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_Simulator">
        <property name="text">
         <string>Simulator:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QRadioButton" name="rb_Incremental">
        <property name="toolTip">
         <string>add to existing set</string>
        </property>
        <property name="text">
         <string>incremental</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QRadioButton" name="rb_DbIcaoCodesOnly">
        <property name="toolTip">
         <string>Model has ICAO code known in DB</string>
        </property>
        <property name="text">
         <string>DB  ICAO data</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QRadioButton" name="rb_WithIcaoData">
        <property name="toolTip">
         <string>Model has ICAO code, but not necessarily valid data</string>
        </property>
        <property name="text">
         <string> ICAO data</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QCheckBox" name="cb_ConsolidateModelSet">
        <property name="text">
         <string>consolidate with DB</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbl_SelectSourceSet">
        <property name="text">
         <string>Source set:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="BlackGui::Components::CSimulatorSelector" name="comp_SimulatorSelector">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="rb_DbDataOnly">
        <property name="toolTip">
         <string>Own models for which DB data are available</string>
        </property>
        <property name="text">
         <string>DB data only</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbl_Providers">
        <property name="text">
         <string>Distributors:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QRadioButton" name="rb_DistributorsSelected">
        <property name="text">
         <string>selected</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QRadioButton" name="rb_DisplayPreferencesDistributors">
        <property name="text">
         <string>from preferences</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QRadioButton" name="rb_DistributorsFromBelow">
        <property name="text">
         <string>all from below</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QRadioButton" name="rb_DistributorsAll">
        <property name="text">
         <string>all</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QRadioButton" name="rb_DisplayAllDistributors">
        <property name="text">
         <string>all distributors</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QCheckBox" name="cb_SortByPreferences">
        <property name="toolTip">
         <string>If checked the distributors from the settings page will be used</string>
        </property>
        <property name="text">
         <string>sort by dist. preferences</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gb_Distributors">
     <property name="title">
      <string>Distributors</string>
     </property>
     <layout class="QVBoxLayout" name="vl_Distributors">
      <item>
       <widget class="BlackGui::Views::CDistributorView" name="tvp_Distributors">
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="showGrid">
         <bool>false</bool>
        </property>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlackGui::Components::CSimulatorSelector</class>
   <extends>QFrame</extends>
   <header>blackgui/components/simulatorselector.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlackGui::Views::CDistributorView</class>
   <extends>QTableView</extends>
   <header>blackgui/views/distributorview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="bg_SourceSet"/>
  <buttongroup name="bg_Distributors"/>
  <buttongroup name="bg_Mode"/>
  <buttongroup name="bg_DisplayedDistributors"/>
 </buttongroups>
</ui>
